### PR TITLE
add third party notice

### PR DESCRIPTION
**Pull request by gsanchezgavier**
- Adds third party notice.
- Adds a CI check to test if is up to date.

### CODE REVIEW

File: .github/workflows/push_pr.yaml [hunk 1]
```diff
@@ -158,6 +158,17 @@ jobs:
...
+      # some of the indirect dependencies.
+      # - name: Check THIRD_PARTY_NOTICES.md is up to date
+      #   run: |
+      #     go install go.elastic.co/go-licence-detector@v0.5.0
+      #     make build-license-notice
+      #     if [ ! -z "$(git status --porcelain)" ]; then 
+      #       git diff
+      #       exit 1
+      #     fi
+
```
Comment (gsanchezgavier) [inline .github/workflows/push_pr.yaml:171]: i couldn't make it work 😢  but the idea is there, if we could generate this each time and check if outdated.

Comment (kang-makes) [reply .github/workflows/push_pr.yaml:171]: We could automate it in another PR. No worries.

**Review by kang-makes — APPROVED** [review]
I left a comment but I see no issue to not approve it!

Congrats! Really great work!

File: assets/licence/overrides [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{"name": "github.com/chzyer/logex", "licenceType": "MIT"}
```
[the remark this answers is not in the document]
Comment (gsanchezgavier) [reply assets/licence/overrides:1]: the tool doesn't pick them automatically for some reason i didn't dig

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+All notable changes are documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+## [0.0.1] - 2022-09-20
+### Added
+- First Version of `newrelic-prometheus-configurator`.
```
[the remark this answers is not in the document]
Comment (gsanchezgavier) [reply CHANGELOG.md:11]: nope this was just a manual changelog i created

**Pull request merged**